lib/spack/spack/binary_distribution.py

```diff
@@ -782,6 +849,7 @@ def generate_package_index(cache_prefix):
         msg = 'Encountered problem pushing package index to {0}: {1}'.format(
             cache_prefix, err)
         tty.warn(msg)
+        traceback.print_exc(file=sys.stdout)
```

Comment:
This looks like it's vestigial debugging code.

Reply:
Well, if debug is on, I'd love to be able to print the whole stack trace.  What's the canonical way to do that?